com.unity.netcode.gameobjects/package.json

```diff
@@ -8,5 +8,10 @@
         "com.unity.modules.ai": "1.0.0",
         "com.unity.modules.animation": "1.0.0",
         "com.unity.nuget.mono-cecil": "1.10.1-preview.1"
-    }
+    },
+	"samples": [{        
+         "displayName": "Bootstrap Template",        
+         "description": "Lightweight template for getting started with Netcode for GameObjects.",        
+         "path": "Samples~/Bootstrap"
+    }]
```

Comment:
I think this should be removed and added by UPM automatically as we discussed earlier.

Reply:
Done.